src/config.demo.ts

```diff
@@ -0,0 +1,10 @@
+/*
+* Clone this file and remove demo
+* you should have config.ts
+* then replace 'API_URL' with your local server
+* and 'PUSHER_KEY' with a valid value
+*/
+export const env = {
+    API_URL:'http://192.168.1.12:3000',
+    PUSHER_KEY:'REPPLACE-WITH-YOUR-KEY'
```

Comment:
This is probably the most important part of this commit, since we don't have `babel-plugin-transform-inline-environment-variables` now for transforming the config variables and pass them to the app, I created this config file, the first time we setup the project the first time we only need to clone this file with the name `config.ts` and replace these variables with the valid values.